### PR TITLE
Update the video editor UI [develop]

### PR DESCRIPTION
**Pull request by Juzar10**
- Update the video editor UI with latest changes.
- Update the styling and refactoring of the code.

### CODE REVIEW

File: inc/classes/class-assets.php [hunk 1]
```diff
@@ -212,6 +212,7 @@ public function admin_enqueue_scripts() {
 				'nonce'                    => wp_create_nonce( 'easydam_media_library' ),
 				'godamToolsNonce'          => wp_create_nonce( 'rtgodam_tools' ),
 				'enableFolderOrganization' => $enable_folder_organization,
+				'isPollPluginActive' => is_plugin_active( 'wp-polls/wp-polls.php' ),
```
Comment (rtBot) [inline inc/classes/class-assets.php:215]: :warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&#039;isPollPluginActive&#039;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

File: inc/classes/class-pages.php [hunk 2]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
 					array(),
 					'1.0.0'
 				);
-			}       
+			}
+
+			$poll_ajax_style = get_option('poll_ajax_style');
```
Comment (rtBot) [inline inc/classes/class-pages.php:339]: :no_entry_sign: **Error**: Expected 1 spaces after opening parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:339]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

File: inc/classes/class-pages.php [hunk 3]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+
+			if ( is_plugin_active( 'wp-polls/wp-polls.php' ) && isset( $poll_ajax_style['loading'] ) && $poll_ajax_style['loading'] ) {
+
+				if ( ! defined( 'WP_POLLS_VERSION' ) ) {
+					define( 'WP_POLLS_VERSION', '2.77.3' );
+				}
+
+				wp_enqueue_script('wp-polls', plugins_url('wp-polls/polls-js.js'), array('jquery'), WP_POLLS_VERSION, true);
```
Comment (rtBot) [inline inc/classes/class-pages.php:347]: :no_entry_sign: **Error**: Expected 1 spaces after opening parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:347]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:347]: :no_entry_sign: **Error**: Expected 1 space after the array opener in a single line array. Found: no spaces (*NormalizedArrays.Arrays.ArrayBraceSpacing.SpaceAfterArrayOpenerSingleLine*).

Comment (rtBot) [inline inc/classes/class-pages.php:347]: :no_entry_sign: **Error**: Expected 1 space before the array closer in a single line array. Found: no spaces (*NormalizedArrays.Arrays.ArrayBraceSpacing.SpaceBeforeArrayCloserSingleLine*).

File: inc/classes/class-pages.php [hunk 4]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+				wp_enqueue_style('wp-polls', plugins_url('wp-polls/polls-css.css'), false, WP_POLLS_VERSION, 'all');
```
Comment (rtBot) [inline inc/classes/class-pages.php:348]: :no_entry_sign: **Error**: Expected 1 spaces after opening parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:348]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

File: inc/classes/class-pages.php [hunk 5]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+
+				wp_localize_script('wp-polls', 'pollsL10n', array(
```
Comment (rtBot) [inline inc/classes/class-pages.php:350]: :no_entry_sign: **Error**: Opening parenthesis of a multi-line function call must be the last content on the line (*PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket*).

**Review by rtBot — CHANGES_REQUESTED** [review]
# Code analysis identified issues
[action-phpcs-code-review](https://github.com/rtCamp/action-phpcs-code-review/) has identified potential problems in this pull request during automated scanning. We recommend reviewing the issues noted and that they are resolved.

**phpcs** scanning turned up:

:no_entry_sign: 28 errors 

:warning: 9 warnings 


***


Powered by rtCamp's [GitHub Actions Library](https://github.com/rtCamp/github-actions-library)



Posting will continue in further review(s)

File: inc/classes/class-pages.php (hunk 5 above)
Comment (rtBot) [inline inc/classes/class-pages.php:350]: :no_entry_sign: **Error**: Only one argument is allowed per line in a multi-line function call (*PEAR.Functions.FunctionCallSignature.MultipleArguments*).

File: inc/classes/class-pages.php [hunk 6]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+					'ajax_url' => admin_url('admin-ajax.php'),
```
Comment (rtBot) [inline inc/classes/class-pages.php:351]: :warning: **Warning**: Array double arrow not aligned correctly; expected 6 space(s) between &quot;&#039;ajax_url&#039;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (rtBot) [inline inc/classes/class-pages.php:351]: :no_entry_sign: **Error**: Expected 1 spaces after opening parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:351]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

File: inc/classes/class-pages.php [hunk 7]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+					'text_wait' => __('Your last request is still being processed. Please wait a while ...', 'wp-polls'),
```
Comment (rtBot) [inline inc/classes/class-pages.php:352]: :warning: **Warning**: Array double arrow not aligned correctly; expected 5 space(s) between &quot;&#039;text_wait&#039;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (rtBot) [inline inc/classes/class-pages.php:352]: :no_entry_sign: **Error**: Expected 1 spaces after opening parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:352]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

**Review by rtBot — CHANGES_REQUESTED** [review]
Previous scan continued.

File: inc/classes/class-pages.php [hunk 8]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+					'text_valid' => __('Please choose a valid poll answer.', 'wp-polls'),
```
Comment (rtBot) [inline inc/classes/class-pages.php:353]: :no_entry_sign: **Error**: Mismatched text domain. Expected &#039;godam&#039; but got &#039;wp-polls&#039; (*WordPress.WP.I18n.TextDomainMismatch*).

Comment (rtBot) [inline inc/classes/class-pages.php:353]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

File: inc/classes/class-pages.php [hunk 9]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+					'text_multiple' => __('Maximum number of choices allowed: ', 'wp-polls'),
```
Comment (rtBot) [inline inc/classes/class-pages.php:354]: :no_entry_sign: **Error**: Expected 1 spaces after opening parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (rtBot) [inline inc/classes/class-pages.php:354]: :no_entry_sign: **Error**: Mismatched text domain. Expected &#039;godam&#039; but got &#039;wp-polls&#039; (*WordPress.WP.I18n.TextDomainMismatch*).

Comment (rtBot) [inline inc/classes/class-pages.php:354]: :no_entry_sign: **Error**: Expected 1 spaces before closing parenthesis; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

File: inc/classes/class-pages.php [hunk 10]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+					'show_loading' => (int) $poll_ajax_style['loading'],
```
Comment (rtBot) [inline inc/classes/class-pages.php:355]: :warning: **Warning**: Array double arrow not aligned correctly; expected 2 space(s) between &quot;&#039;show_loading&#039;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

File: inc/classes/class-pages.php [hunk 11]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+					'show_fading' => (int) $poll_ajax_style['fading']
```
Comment (rtBot) [inline inc/classes/class-pages.php:356]: :warning: **Warning**: Array double arrow not aligned correctly; expected 3 space(s) between &quot;&#039;show_fading&#039;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (rtBot) [inline inc/classes/class-pages.php:356]: :no_entry_sign: **Error**: There should be a comma after the last array item in a multi-line array (*NormalizedArrays.Arrays.CommaAfterLast.MissingMultiLine*).

File: inc/classes/class-pages.php [hunk 12]
```diff
@@ -334,7 +334,28 @@ public function admin_enqueue_scripts() {
...
+				));
```
Comment (rtBot) [inline inc/classes/class-pages.php:357]: :no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).

File: inc/classes/rest-api/class-polls.php [hunk 13]
```diff
@@ -0,0 +1,110 @@
+<?php
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-bqh4sr.php, but found vipgoci-phpcs-scan-BQh4SR.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

**Review by rtBot — CHANGES_REQUESTED** [review]
Previous scan continued.

File: inc/classes/rest-api/class-polls.php (hunk 13 above)
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:1]: :no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-polls.php, but found vipgoci-phpcs-scan-BQh4SR.php (*WordPress.Files.FileName.InvalidClassFileName*).

File: inc/classes/rest-api/class-polls.php [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+				'namespace' => $this->namespace,
+				'route'     => '/' . $this->rest_base . '/poll/(?P<id>\d+)',
+				'args'      => array(
+					array(
+						'methods'             => \WP_REST_Server::READABLE,
+						'callback'            => array( $this, 'get_poll' ),
+						'permission_callback' => '__return_true',
+						'args'                => 
+							array(
+								'id'    => array(
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:41]: :warning: **Warning**: Array double arrow not aligned correctly; expected 1 space(s) between &quot;&#039;id&#039;&quot; and double arrow, but found 4 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

File: inc/classes/rest-api/class-polls.php [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+									'description'       => 'The ID of the Poll.',
+									'type'              => 'integer',
+									'required'          => true,
+									'sanitize_callback' => 'absint',
+								)
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:46]: :no_entry_sign: **Error**: There should be a comma after the last array item in a multi-line array (*NormalizedArrays.Arrays.CommaAfterLast.MissingMultiLine*).

File: inc/classes/rest-api/class-polls.php [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+							),
+						),
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:48]: :no_entry_sign: **Error**: Array closer not aligned correctly; expected 20 space(s) but found 24 (*WordPress.Arrays.ArrayIndentation.CloseBraceNotAligned*).

File: inc/classes/rest-api/class-polls.php [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+	 * @return \WP_REST_Response
+	 */
+	public function get_polls() {
+		global $wpdb;
+
+		if ( ! $this->is_poll_plugin_active() ) {
+			return new \WP_Error( 'poll_plugin_not_active', 'Poll plugin is not active.', array( 'status' => 404 ) );
+		}
+
+		$polls = $wpdb->get_results( "SELECT * FROM $wpdb->pollsq ORDER BY pollq_timestamp DESC" );
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:66]: :warning: **Warning**: Use of a direct database call is discouraged (*WordPress.DB.DirectDatabaseQuery.DirectQuery*).

Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:66]: :warning: **Warning**: Direct database call without caching detected. Consider using wp_cache_get() / wp_cache_set() or wp_cache_delete() (*WordPress.DB.DirectDatabaseQuery.NoCaching*).

File: inc/classes/rest-api/class-polls.php [hunk 18]
```diff
@@ -0,0 +1,110 @@
...
+
+		return rest_ensure_response( $polls );
+	}
+
+	/**
+	 * Get a single Poll.
+	 *
+	 * @param \WP_REST_Request $request
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:74]: :no_entry_sign: **Error**: Missing parameter comment (*Squiz.Commenting.FunctionComment.MissingParamComment*).

**Review by rtBot — CHANGES_REQUESTED** [review]
Previous scan continued.

File: inc/classes/rest-api/class-polls.php [hunk 19]
```diff
@@ -0,0 +1,122 @@
+<?php
```
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-hwqz4z.php, but found vipgoci-phpcs-scan-hWQZ4z.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:1]: :no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-polls.php, but found vipgoci-phpcs-scan-hWQZ4z.php (*WordPress.Files.FileName.InvalidClassFileName*).

**Review by rtBot — CHANGES_REQUESTED** [review]
# Code analysis identified issues
[action-phpcs-code-review](https://github.com/rtCamp/action-phpcs-code-review/) has identified potential problems in this pull request during automated scanning. We recommend reviewing the issues noted and that they are resolved.

**phpcs** scanning turned up:

:no_entry_sign: 2 errors 


***


Powered by rtCamp's [GitHub Actions Library](https://github.com/rtCamp/github-actions-library)

File: inc/classes/rest-api/class-polls.php (hunk 19 above)
Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-hvoqda.php, but found vipgoci-phpcs-scan-HVOQDa.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

Comment (rtBot) [inline inc/classes/rest-api/class-polls.php:1]: :no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-polls.php, but found vipgoci-phpcs-scan-HVOQDa.php (*WordPress.Files.FileName.InvalidClassFileName*).

**Review by rtBot — CHANGES_REQUESTED** [review]
# Code analysis identified issues
[action-phpcs-code-review](https://github.com/rtCamp/action-phpcs-code-review/) has identified potential problems in this pull request during automated scanning. We recommend reviewing the issues noted and that they are resolved.

**phpcs** scanning turned up:

:no_entry_sign: 2 errors 


***


Powered by rtCamp's [GitHub Actions Library](https://github.com/rtCamp/github-actions-library)

File: pages/godam/components/tabs/GeneralSettings/GeneralSettings.jsx [hunk 20]
```diff
@@ -20,7 +20,7 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import BrandImageSelector from './BrandImageSelector.jsx';
-import ColorPickerButton from '../../../../video-editor/components/ColorPickerButton.js';
+import ColorPickerButton from '../../../../video-editor/components//shared/color-picker/ColorPickerButton.jsx';
```
Comment (KMchaudhary) [inline pages/godam/components/tabs/GeneralSettings/GeneralSettings.jsx:23]: Check the double slash

File: pages/video-editor/VideoEditor.js [hunk 21]
```diff
@@ -252,15 +214,6 @@ const VideoEditor = ( { attachmentID } ) => {
 							</div>
 						</div>
 					) }
-
-					{
-						loading && (
```
Comment (KMchaudhary) [inline pages/video-editor/VideoEditor.js:256]: Video loading skeleton is not required now?

**Review by KMchaudhary — APPROVED** [review]

File: pages/godam/components/tabs/GeneralSettings/GeneralSettings.jsx (hunk 20 above)
Comment (Juzar10) [reply pages/godam/components/tabs/GeneralSettings/GeneralSettings.jsx]: @KMchaudhary, thanks for noticing that. I have fixed that. It will still working with double slash, not sure how.
